azure/packages/azure-service-utils/package.json

```diff
@@ -44,8 +44,8 @@
 			}
 		}
 	},
-	"main": "./dist/index.js",
-	"types": "./dist/public.d.ts",
+	"main": "lib/index.js",
+	"types": "lib/index.d.ts",
```

Comment:
This became untrimmed. (Not going looking for more.)

Reply:
Thank you very much for catching that.  I'm embarrassed I didn't see it.